src/app/components/Notification/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { createContext, useContext, useEffect, useState } from 'react'
+import { StatusType, Notification } from 'grommet'
+
+interface NotificationContextProps {
+  readonly notificationVisible: boolean
+  readonly notificationTitle: string
+  readonly notificationStatus: StatusType
+  showNotification(message: string, type?: StatusType): void
+  hideNotification(): void
+}
+
+interface NotificationProviderProps {
+  children: React.ReactNode
+}
+
+const NotificationContext = createContext<NotificationContextProps>({} as NotificationContextProps)
+
+export const NotificationProvider = (props: NotificationProviderProps) => {
+  const [notificationVisible, setNotificationVisible] = useState(false)
+  const [notificationTitle, setNotificationTitle] = useState('')
+  const [notificationStatus, setNotificationStatus] = useState<StatusType>('normal')
+  const showNotification = (title: string, status: StatusType = 'normal') => {
+    setNotificationVisible(true)
+    setNotificationTitle(title)
+    setNotificationStatus(status)
+  }
+  const hideNotification = () => setNotificationVisible(false)
+
+  useEffect(() => {
+    if (notificationVisible) {
+      const timeout = setTimeout(hideNotification, 2000)
+      return () => clearTimeout(timeout)
```

Comment:
This is reimplements Grommet's autoClose and notification.toast.time

Reply:
Right. Fixed that. (The reason I didn't notice this was that the default closing time was 8 secs. Too long, I never waited that long.)